frontend/mr-admin-flate/src/components/avtaler/opprett/OpprettAvtaleContainer.tsx

```diff
@@ -132,6 +158,25 @@ export function OpprettAvtaleContainer({
     }
   };
 
+  const sjekkOrgnr = async (orgnr: string) => {
+    virksomhetDispatcher({ type: "Reset" });
+    if (orgnr.trim().length === 9) {
+      virksomhetDispatcher({ type: "Hent data" });
+      const response =
+        await mulighetsrommetClient.hentVirksomhet.hentVirksomhetMedOrgnr({
+          orgnr: orgnr.trim(),
+        });
+
+      if (response) {
+        virksomhetDispatcher({ type: "Data hentet", payload: response });
+      } else {
+        setError("leverandor", {
+          message: `Fant ikke leverandør med nummer: ${orgnr}`,
+        });
```

Comment:
Bruker React-hook-form sin egen `setError` for å koble feil opp til validering av skjema. Dermed trengs ikke en egen error-håndtering i `virksomhetDispatcher`.

Reply:
Er dette noe som evt. kunne passet til å bli trukket ut i egen `OrganisjonsnummerInput`-komponent (abstraksjon over @navikt/ds-react sin TextField-komponent, men med custom validering) gjenbruk i andre skjema?

Reply:
Ja, god idé.